### ISSUE TITLE
Undefined method map for nil when `config.active_record.query_log_tags_enabled` is true

### ISSUE THREAD

**Issue by MaksJS** [rails/solid_cache#117]
On Rails 7.1, when setting `config.active_record.query_log_tags_enabled` to true, I get this error:

`undefined method `map' for nil`

```ruby
        def type_casted_binds(binds)
          binds.map do |value|
            if ActiveModel::Attribute === value
              type_cast(value.value_for_database)
            else
```

in `[activerecord (7.1.2) lib/active_record/connection_adapters/abstract/quoting.rb:235:in `type_casted_binds']`

We always go into this line:

```ruby
 result = connection.select_all(sanitize_sql([ query, values ]), "#{name} Load", nil, preparable: false)
```

and `binds` are `nil` because `connection.prepare_statements?` is false, due to https://github.com/rails/rails/blob/581074c40d0989ff1d981086043b40dc8daa4888/activerecord/lib/active_record/railtie.rb#L422

**Comment by MaksJS** [comment]
Ok I see there's already a pending fix: https://github.com/rails/solid_cache/pull/112

**Comment by djmb** [comment]
@MaksJS - I've merged that fix into main - could you check if it resolves this for you? 🙏

**Comment by MaksJS** [comment]
@djmb Yes it does 🎉 Thanks a lot. Could you release a new version?

**Comment by djmb** [comment]
Hi @MaksJS - I've just released v0.3.0.
